src/harshadBJun20/Assignment22/CheckStringPalindromeOrNot.java

```diff
@@ -0,0 +1,29 @@
+package harshadBJun20.Assignment22;
+
+/*Program 2:
+Verify given String is palindrome or not. 
+Input: radar
+Output : Given string radar is palindrome.
+
+Input: technocredits
+Output: Given string technocredits is not palindrome.*/
+
+public class CheckStringPalindromeOrNot {
+	
+	void checkPalindrome(String str) {
+		String str1="";
+		for(int i=str.length()-1;i>=0;i--) {
```

Comment:
Please use variable name as index, rest the code is perfect!

Reply:
Ok.. I will use index as variable in future prog.